galette/templates/default/elements/navigation/navigation_aside.html.twig

```diff
@@ -25,7 +25,9 @@
 
 {% if not login.getCompactMenu() %}
     <div class="ui basic center aligned fitted segment">
+<a href="https://{{ preferences.pref_footer|raw }}">
```

Comment:
I guess you meant:
```suggestion
<a href="https://{{ preferences.pref_website|raw }}">
```

But also, we certainly need to check if this value is not empty